op-chain-ops/genesis/layer_one_test.go

```diff
@@ -54,8 +54,6 @@ func TestBuildL1DeveloperGenesis(t *testing.T) {
 
```

Comment:
This file tests against an old state dump. Its not super helpful at this point to be honest, we have real test coverage in the deploy script with chain assertions